scripts/custom_lint_checks.py

```diff
@@ -269,6 +269,14 @@ def visit_functiondef(self, node):
         self.check_functiondef_yields(node, node_doc)
 
     def check_functiondef_params(self, node, node_doc):
+        """Checks whether all parameters in a function definition are documented.
+
+        Args:
+            node: astroid.scoped_nodes.Function. Node for a function or
+                method definition in the AST.
+            node_doc: Pylint docstring class. It represents an AST node's
```

Comment:
Are you sure that this is the correct class type? Ditto below for all `node_doc` usage.

Reply:
@apb7 I checked this up. So, all *_doc variables are of the type(_checker_docs_utils.Docstring) class. This is a pylint class.

Reply:
Great, thanks @pranavsid98!